jina/hubble/__init__.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class HubExecutor:
+    """Basic Executor Data Class from Hubble"""
+
+    uuid: str = None
+    alias: Optional[str] = None
+    sn: Optional[int] = None
+    tag: Optional[str] = None
```

Comment:
To my understanding, `tag` is always assigned automatically.
```suggestion
    tag: str = None
```